### ISSUE TITLE
[BUG] Parser/Linter error `function randomstr wants argument count 1 but provides 1`

### ISSUE THREAD

**Issue by chenrui333** [ysugimoto/falco#33]
Looks like a false alarm with the following error report?

```
🔥 [ERROR] function randomstr wants argument count 1 but provides 1 (function/arguments)
polyfill-service/fastly/vcl/fastly-boilerplate-begin.vcl at line 4, position 54
 3|    if (!req.http.X-Request-Id) {
 4|        set req.http.X-Request-Id = digest.hash_sha256(now randomstr(64) req.http.host req.url req.http.Fastly-Client-IP server.identity);
                                                              ^^^^^^^^^
 5|    }
See document in detail: https://developer.fastly.com/reference/vcl/functions/randomness/randomstr/
```

**Comment by ysugimoto** [comment]
@chenrui333

Thanks, the second argument of `randomstr` is optional [document](https://developer.fastly.com/reference/vcl/functions/randomness/randomstr/) so it's a bug.

I'll fix it later.

**Comment by ysugimoto** [comment]
@chenrui333 I've fixed this problem in #35 and released `v.0.6.0`.
Please use the latest version.
